test/message_bus/message_outbox/outboxMessageHash.js

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 OpenST Ltd.
+//
```

Comment:
Change the file name to `outbox_message_hash.js`

Reply:
Done.